lib/screens/downloads_settings_screen.dart

```diff
@@ -37,52 +37,51 @@ class DownloadsSettingsScreen extends StatelessWidget {
           ),
           if (Platform.isIOS || Platform.isAndroid) const RequireWifiSwitch(),
           const ShowPlaylistSongsSwitch(),
+          const SyncFavoritesSwitch(),
+          ListTile(
+            // TODO real UI for this
+            title: const Text("Show all playlists offline"),
+            subtitle: const Text(
+                "Sync metadata for all playlists to show partially downloaded playlists offline"),
+            trailing: DownloadButton(
+                infoOnly: true,
+                item: DownloadStub.fromFinampCollection(
+                    FinampCollection(type: FinampCollectionType.allPlaylists))),
+          ),
```

Comment:
This prompts for transcoding options (and probably also download location), which doesn't make sense

Reply:
Actually, we need a download location for the images this generates.  I'll make it skip transcoding prompts though.

Reply:
It's still/again asking for transcoding settings with the latest commits ^^